### DISCUSSION TITLE
Tactical Theme and Opening list entries in alphabetical order?

### DISCUSSION THREAD

**Discussion by rodpolako** [brianch/offline-chess-puzzles#21] [Ideas]
First off, let me just say that I love the app.  It is a really cool idea and works great.

I hate to even bring this up but one minor thing that bugs me is that if I wanted to search for a specific theme or opening I end up spending a bit of time just trying to find it and if I don't see it I'm not sure if it was because I missed it or that it doesn't exist.  

If sorting the list alphabetically isn't feasible or there is a reason for the order that they are displayed in, would it be possible to make the dropdown allow me to type what I'm looking for and have it shorten the list to the one's that match?

For instance, say I was looking for puzzles in the King's Indian opening, I think it would be nice to either be able to scroll the list to those starting with "K" or if I was able to start typing "king" then the list would shorten to show King's Gambit Accepted, King's Pawn Game, King's Indian, etc.

Just a thought.  Thanks for reading.

**Comment by brianch** [comment]
Hi!

Yeah, I agree it's a bit annoying, currently it's in the same order that we have here:
https://lichess.org/training/openings (So it's sorted by number of puzzles in that opening)

In the way it's implemented now, I can make it alphabetical in English, but the translated versions wouldn't be, so I'd need to check if it can be done in some other way or if it's possible to implement a filter, as you said.

I'll try to investigate this a bit more when I have time, thanks for the suggestions!
